config/rbac/role.yaml

```diff
@@ -110,3 +110,4 @@ rules:
   verbs:
   - get
   - list
+  - watch
```

Comment:
This file is autogenerated, so this should be set in a comment inside `api` package